src/Config/Loader/XliffFileLoader.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Config\Loader;
+
+use Symfony\Component\Config\Loader\Loader;
+
+/**
+ * XliffFileLoader reads XLIFF files and converts them to Contao language array
+ *
+ * @author Andreas Schempp <https://github.com/aschempp>
+ * @author Leo Feyer <https://github.com/leofeyer>
+ */
+class XliffFileLoader extends Loader
+{
+    private $rootDir;
+    private $addToGlobals;
+
+    /**
+     * Constructor.
+     *
+     * @param string $rootDir      The kernel root directory
+     * @param bool   $addToGlobals Defines if language labels should be added to $GLOBALS['TL_LANG']
+     */
+    public function __construct($rootDir, $addToGlobals = false)
+    {
+        $this->rootDir      = dirname($rootDir);
+        $this->addToGlobals = $addToGlobals;
+    }
+
+    /**
+     * Read the contents of a PHP file, stripping the opening and closing PHP tags
+     *
+     * @param string      $file A PHP file path
+     * @param string|null $type The resource type
+     *
+     * @return string The PHP code without the PHP tags
+     */
+    public function load($file, $type = null)
+    {
+        $language = $type ?: 'en';
```

Comment:
Because the interface defines this variable, and it can be used for anything suitable for the loader.